solutions/instances/variables.tf

```diff
@@ -43,6 +43,151 @@ variable "prefix" {
   default     = null
 }
 
+##############################################################################
+# IBM Cloud Logs
+##############################################################################
+variable "cloud_logs_provision" {
+  description = "Provision a IBM Cloud Logs instance?"
+  type        = bool
+  default     = false
```

Comment:
why do we have default value as `false`? For log analysis, monitoring..default is `true`.

Reply:
@iamar7 I think you missed this comment. Set this to true, and logdna to false now

Reply:
@iamar7 I think you missed this comment. Set this to true, and logdna to false now